onchain/rollups/test/foundry/dapp/helper/.gitignore

```diff
@@ -1,5 +1 @@
-gen-proofs.sh
-finish_epoch_response.json
 output
```

Comment:
this is minor. We could add `input` here and remove the `.gitignore` file in the input folder

Reply:
So, I've intentionally left `input` committed with a `.gitignore`, because, otherwise, `docker` would create this folder with `root` permissions, and `forge` wouldn't be able to write to files in it. By leaving `input` committed, we automatically assign user permissions to it.